Resources/Prototypes/_ES/Catalog/Fills/wardrobe_fun.yml

```diff
@@ -0,0 +1,148 @@
+- type: entity
+  parent: ESWardrobeTheatre
+  id: ESWardrobeTheatreFilled
+  suffix: Filled
+  components:
+  - type: EntityTableContainerFill
+    containers:
+      entity_storage: !type:NestedSelector
+        tableId: ESFillWardrobeTheatre
+
+- type: entityTable
+  id: ESFillWardrobeTheatre
```

Comment:
this is an insane way to do this. Just make this a group selector with all the outfits and make it roll 3-5 times.

also you can removed all instances of nested selector and the RangeNumberSelector.

you can also remove AllSelector by replacing "children" with "all:"